client/src/Components/Home/Home.css

```diff
@@ -12,13 +12,13 @@
   align-items: center;
   text-align: center;
   justify-content: center;
-  border: 1px solid white;
+  border: 2px solid white;
```

Comment:
yes! thank you, this is something I was thinking about as a second order adjustment.